### PR TITLE
Prepare release for Azure.ResourceManager.DnsResolver 1.2.0-beta.1

### PR DESCRIPTION
**Pull request by ArthurMa1978**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
 # Release History
 
-## 1.2.0-beta.1 (2024-10-18)
+## 1.2.0-beta.1 (2024-10-24)
 
 ### Features Added
 
-- Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
+- Upgraded api-version tag from 'package-2022-07' to 'package-preview-2023-07'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/b26a190235f162b15d77dad889d104d06871fb4f/specification/dnsresolver/resource-manager/readme.md.
+    - Add DNS Security Policy functionality for the following resources:
```
Comment (live1206) [inline sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md:8]: why is this a sub-item under the api-version bump?

File: sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md [hunk 2]
```diff
@@ -1,22 +1,28 @@
 # Release History
 
-## 1.2.0-beta.1 (2024-10-18)
+## 1.2.0-beta.1 (2024-10-24)
 
 ### Features Added
 
-- Enable the new model serialization by using the System.ClientModel, refer this [document](https://aka.ms/azsdk/net/mrw) for more details.
+- Upgraded api-version tag from 'package-2022-07' to 'package-preview-2023-07'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/b26a190235f162b15d77dad889d104d06871fb4f/specification/dnsresolver/resource-manager/readme.md.
```
Comment (live1206) [inline sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md:7]: do we need to specify the tag detail in the changelog later on?

Comment (ArthurMa1978) [reply sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md:7]: No, we have discussed before that don't want to have details of api change in changelog.

**Review by live1206 — APPROVED** [review]

**Comment by jamesvoongms** [comment]
Looks good to me, thank you

**Pull request merged**